packages/bits/spec/components/dialog/dialog.atom.ts

```diff
@@ -41,4 +42,10 @@ export class DialogAtom extends Atom {
     public scrollToEnd = async (): Promise<void> => browser.executeScript<void>(`arguments[0].scrollIntoView({block: "end"})`, this.getDialog());
 
     public getDialog(): ElementFinder { return super.getElement().element(by.className("modal-dialog")); }
+
+    public getDatetimePicker(): DateTimepickerAtom {
```

Comment:
We should avoid adding this method to the dialog atom since a datetime picker isn't a basic/normal part of a dialog.